dask/tests/test_distributed.py

```diff
@@ -659,3 +659,27 @@ async def test_pack_MaterializedLayer_handles_futures_in_graph_properly(c, s, a,
     packed = hlg.__dask_distributed_pack__(c, ["z"], {})
     unpacked = HighLevelGraph.__dask_distributed_unpack__(packed)
     assert unpacked["deps"] == {"x": {fut.key}, "y": {fut.key}, "z": {"y"}}
+
+
+@gen_cluster(client=True)
+async def test_to_sql_engine_kwargs(c, s, a, b):
```

Comment:
If my earlier question about `engine_kwargs` not being needed is true, then we can probably update this test name accordingly too 

```suggestion
async def test_to_sql(c, s, a, b):
```

Reply:
They aren't needed but I think they are part of what we want to test?

Reply:
Okay, sounds good. I only skimmed the original issue, so if it's meaningful to keep the kwargs, then let's keep them